### PR TITLE
Add sabre templates.

### PR DESCRIPTION
**Pull request by nickmbailey**
Our spreedly account email is 'dev@travelboss.ai' and our environment key is 19N8Fv0740kZ1WpEYUtA1cKCOUG

### CODE REVIEW

File: receivers/sabre/json/flight_book_deliver.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+        "headers": "Authorization: Bearer SABRETOKEN\nApplication-ID: SWS1:SBR-CtntSerLgUI:1f06045fcf\nAccept: application/json\nAcceptEncoding: gzip\nContent-Type: application/json",
+        "payment_method_token": "LNHjjMBXy9NIYGYGqwKy1L1grXC",
```
Comment (dlj154) [inline receivers/sabre/json/flight_book_deliver.json]: Replace token value with `REPLACE_WITH_PAYMENT_METHOD_TOKEN`

File: receivers/sabre/json/hotel_book_deliver.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        "headers": "Authorization: Bearer SABRETOKEN\nApplication-ID: SWS1:SBR-CtntSerLgUI:1f06045fcf\nAccept: application/json\nAcceptEncoding: gzip\nContent-Type: application/json",
+        "payment_method_token": "LNHjjMBXy9NIYGYGqwKy1L1grXC",
```
Comment (dlj154) [inline receivers/sabre/json/hotel_book_deliver.json]: Replace token value with `REPLACE_WITH_PAYMENT_METHOD_TOKEN`

**Comment by dlj154** [comment]
Thank you for contributing, @nickmbailey! Very excited to add another template to our new library. 

After reviewing, we decided it's probably best not to include actual tokens in the templates. I left comments on the two lines. Would you mind making those changes and then this will be good to go? Thank you, and if you have any feedback please let me know!

**Comment by nickmbailey** [comment]
Sure thing. Should be good now. 👍

**Review by dlj154 — APPROVED** [review]
Looks great! Thank you so much for contributing!

**Pull request merged**